ProactivePolicyTemplate/src/Polly.Contrib.ProactiveCustomPolicyTemplate.Specs/Polly.Contrib.ProactiveCustomPolicyTemplate.Specs.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;netcoreapp2.0;net462;net472</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;netcoreapp2.0;</TargetFrameworks>
```

Comment:
This is confusing. Is there a requirement to test under all 4 frameworks? Can net462, and net472 be optional via env variable or something alike?

Reply:
Main Polly repo does not target those frameworks as well https://github.com/App-vNext/Polly/blob/master/src/Polly.NetStandard20.Specs/Polly.NetStandard20.Specs.csproj#L5